src/models/SqlOutputContentProvider.ts

```diff
@@ -275,8 +275,12 @@ export class SqlOutputContentProvider implements vscode.TextDocumentContentProvi
         queryRunner.runQuery(selection);
         let paneTitle = Utils.formatString(Constants.titleResultsPane, queryRunner.title);
         // Always run this command even if just updating to avoid a bug - tfs 8686842
+        this.displayResultPane(resultsUri, paneTitle);
+    }
 
-        // Check if the results window already exists
+    // Intentionally created anonymously for testing purposes
+    public displayResultPane = function(resultsUri: string, paneTitle: string): void {
```

Comment:
This is kinda goofy. I think using TypeMoq you can just mock a real instance of this class and override this one function, avoiding the need to add in a setDisplayResultPane and declare this anonymously. However it's not critical to fix.